### PR TITLE
INTDEV-394 remove buttonLabel and namePrompt

### PR DESCRIPTION
**Pull request by samutoljamo-pareto**
Removed all occurrences of namePrompt/buttonLabel

### CODE REVIEW

File: tools/data-handler/test/command-handler.test.ts [hunk 1]
```diff
@@ -1186,8 +1183,7 @@ describe('create command', () => {
   });
   it('template with "123"', async () => {
     const templateName = 'loc/123';
-    const templateContent =
-      '{"buttonLabel": "Button1", "namePrompt": "Prompt1"}';
+    const templateContent = '{}';
```
Comment (Fuzzbender) [inline tools/data-handler/test/command-handler.test.ts:1186]: As `templateContent` is optional, there could be a test that doesn't pass the `content` parameter at all.
Either a new test, or encapsulate to one of the existing ones.

Comment (Fuzzbender) [reply tools/data-handler/test/command-handler.test.ts:1186]: Actually, if the `content` file is always empty, shouldn't we remove the whole parameter from CLI?

**Review by Fuzzbender — APPROVED** [review]
Check if the content parameter could be removed as unnecessary from CLI.

**Comment by samutoljamo-pareto** [comment]
Its optional so I think we should just leave it. Maybe you'd want to give a displayName while creating the template

File: tools/data-handler/test/command-handler.test.ts (hunk 1 above)
Comment (samutoljamo-pareto) [reply tools/data-handler/test/command-handler.test.ts:1186]: I'll add a new test

**Pull request merged**